about.html

```diff
@@ -0,0 +1,138 @@
+            <p class="contact">wtsummit2023@beatthemarket.com</p>
+        </div>
+    </section>
+
+    <section class="logo-section">
+        <h1>Wintrade Global Summit 2023 Logo</h1>
+        <p class="desktop">Out of numerous submissions, the chosen logo from the competition stands out as a true representation of our brand identity. Its creative design, meaningful symbolism, and harmonious colors make it instantly recognizable and memorable. </p>
+        <p class="mobile">The logo was decided through a logo competition held from Jun 1st to 10th.</p>
+        <div class="box">
+            <img src="./images/logo-removebg-preview.png">
+        </div>
+    </section>
+
+    <section class="history-section">
+        <h1>See the past WT Global Summits</h1>
+        <p>Take a look the summits which were held on London, Sydney and Baku</p>
+        <div class="card-container">
+            <div class="card">
+                <p>2014</p>
+                <p>WT Global Summit in London</p>
+            </div>
+            <div class="card">
+                <p>2017</p>
+                <p>WT Global Summit in Sydney</p>
+            </div>
+            <div class="card">
+                <p>2020</p>
+                <p>WT Global Summit in Baku</p>
+            </div>
+        </div>
+    </section>
+
+    <section class="partners desktop" id="partners">
+        
+        <h1>Partners</h1>
+
+        <div class="sponsors">
+            <img src="./images/airbnb 1.png">
+            <img src="./images/amz.png">
+            <img src="./images/mozilla 1.png">
+            <img src="./images/coke-removebg-preview.png">
+            <img src="./images/tesla.png">
+        </div>
+    </section>
+
+    <!-- Partner Section -->
+    <section class="partners mobile">
+        
+        <h1>Partners</h1>
+
+        <div class="sponsors">
+            <img src="./images/airbnb 1.png">
+            <img src="./images/amz.png">
+            <img src="./images/mozilla 1.png">
+            <img src="./images/coke-removebg-preview.png">
+            <img src="./images/tesla.png">
+        </div>
+    </section>
+
+    <footer>
+        <div class="footer-container">
+          <div class="logo-content">
+            <!-- logo content -->
+            <img src="./images/BeatTheMarket-removebg-preview.png" alt="Logo">
+          </div>
+          <div class="footer-content">
+            <h2>&copy; 2023 BEATtheMARKET. Some rights reserved</h2>
+            <p></p>
+          </div>
+        </div>
+    </footer>
+    <script src="./javascripts/menu-button.js"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+</body>
+</html>
```

Comment:
For the mobile view, kindly implement these design changes :

- Please adjust the position of the mobile menu to the leftmost side of your view, just as it in the design. Please do this for the home page as well 👍 
- Kindly fix this section as it have too much margin and should be similar to our design given, also remember to add the design background color used
Your design :
![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/02e765ae-33cf-46e1-bb49-2adc7655eeb5)

Design given :
![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/0e76c436-93d8-4cbc-9bd4-d457078fa01f)

- Please add a horizontal line to the `partners`title similar to the dsign
- Kindly reduce the font size of the footer so it has similarity with the design given 
-
Your design :
![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/c7b49ed7-f37f-4b4b-b617-a277d585c04b)

Design given :
![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/f69e6c4a-d539-45d7-8fc8-cdf72e1fc2c4)